web/src/components/organisms/Project/Schema/index.tsx

```diff
@@ -0,0 +1,169 @@
+import styled from "@emotion/styled";
+import Layout, { Header, Content } from "antd/lib/layout/layout";
+import Sider from "antd/lib/layout/Sider";
+import React, { useState } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+
+import MoleculeHeader from "@reearth-cms/components/molecules/Common/Header";
+import ProjectMenu from "@reearth-cms/components/molecules/Common/projectMenu";
+import WorkspaceCreationModal from "@reearth-cms/components/molecules/Common/WorkspaceCreationModal";
+import ModelCreationModal from "@reearth-cms/components/molecules/Schema/ModelCreationModal";
+import SchemaMenu from "@reearth-cms/components/molecules/Schema/SchemaMenu";
+
+import useDashboardHooks from "../../Dashboard/hooks";
```

Comment:
If you are going to use other component's hooks in organisms I feel like maybe its okay. But import them in to this component's hooks to keep hooks together

Reply:
@KaWaite , the number of returned props from hooks would be very very large.

Reply:
This comment was missed

Reply:
Either way, having dashboard hooks inside of the schema page isn't really good since they are assigned to the dashboard. 
There is a chance this organism is just doing too much by itself and might need to be split? What do you think?